custom_components/easee/services.yaml

```diff
@@ -48,10 +48,10 @@ set_basic_charge_plan:
       example: "EH123456"
     start_time:
       description: "Plan start time"
-      example: 23:00
+      example: "2020-07-28T19:51"
```

Comment:
How do one go about added a reoccuring schedule every night, is the full date still needed?

Reply:
I think so, you give today's date and include True for repeat. That's how I've understood it at least.